### PR TITLE
Added CI test for generating documentation with sphinx-build

### PR DESCRIPTION
**Pull request by joachimmetz**
Added CI test for generating documentation with sphinx-build

### CODE REVIEW

File: docs/sources/user/Analysis-plugin-virustotal.md [hunk 1]
```diff
@@ -4,19 +4,30 @@ Notes on how to use the virustotal analysis plugin.
 
 ## Getting an API key
 
-The virustotal analysis uses the public Virustotal API, and requires an API key to operate. The process for obtaining an API key is [here](https://www.virustotal.com/en/documentation/public-api/#getting-started).
-
+The virustotal analysis uses the public Virustotal API, and requires an API key
```
Comment (rgayon) [inline docs/sources/user/Analysis-plugin-virustotal.md:7]: VT analysis +plugin

File: docs/sources/developer/Developers-Guide.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ If your code requires adding a new dependency please check out these instruction
 
 ### Before you submit your first code review
 
-1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev)
+1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/#!forum/log2timeline-dev)
```
Comment (rgayon) [inline docs/sources/developer/Developers-Guide.md:69]: In docs/sources/user/Overview.md, the '!' character is changed into %21, not sure whether there is a need to change it here or there

File: docs/sources/user/Analysis-plugin-virustotal.md (hunk 1 above)
Comment (joachimmetz) [reply docs/sources/user/Analysis-plugin-virustotal.md:7]: done

File: docs/sources/developer/Developers-Guide.md (hunk 2 above)
Comment (joachimmetz) [reply docs/sources/developer/Developers-Guide.md:69]: thx for flagging, I'll change it here

**Comment by joachimmetz** [comment]
@rgayon review comments addressed, PTAL

**Pull request merged**